### PR TITLE
Pkgs and functionalities

### PR DESCRIPTION
**Pull request by graiola**
In summary we can say that the ros control framework is composed of various packages and we are going to talk about some of them.

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -74,28 +74,32 @@ bibliography: paper.bib
...
+The repository `ros_controllers` holds several ready-made controllers supporting the most common use-cases for manipulators, mobile and humanoid robots, e.g. the `joint_trajectory_controller` is heavily used with position-controlled robots to interface with MoveIt!.
+
+Finally, `control_msgs` provides ROS messages used in most controllers offered in `ros_controllers`.
 
 `ros_control` was conceptualized by Sachin Chitta at Willow Garage Inc. and initial design and implementation was done by Sachin Chitta (then at Willow Garage), Wim Meussen, Vijay Pradeep and Eitan Marder-Epstein (then at HiDOF) before being released open-source.
 
 `ros_control` is released as binary packages with each new version of ROS, source code is hosted at the [ros-controls](https://github.com/ros-controls) Github organization. Documentation on behaviour, interfaces, doxygen-generated pages and tutorials can be found at [ros_control](http://wiki.ros.org/ros_control) and [ros_controllers](http://wiki.ros.org/ros_controllers). For a thorough presentation we invite the interested reader to watch the talk given at ROSCon2014 [@rodriguez2014roscon].
 
-# Robots using `ros_control`
+# Robots Using `ros_control`
```
Comment (bmagyar) [inline paper.md]: why capitalize?

File: paper.md [hunk 2]
```diff
@@ -74,28 +74,32 @@ bibliography: paper.bib
...
 
-The repository `ros_controllers` holds several ready-made controllers supporting the most common use-cases for manipulators, mobile and humanoid robots, e.g. the `joint_trajectory_controller` is heavily used with position-controlled robots to interface with MoveIt!. Finally, `control_msgs` provides ROS messages used in most controllers offered in `ros_controllers`.
+The repository `ros_controllers` holds several ready-made controllers supporting the most common use-cases for manipulators, mobile and humanoid robots, e.g. the `joint_trajectory_controller` is heavily used with position-controlled robots to interface with MoveIt!.
+
+Finally, `control_msgs` provides ROS messages used in most controllers offered in `ros_controllers`.
```
Comment (bmagyar) [inline paper.md]: please remove this new line

File: paper.md [hunk 3]
```diff
@@ -74,28 +74,32 @@ bibliography: paper.bib
...
 <!-- with some ideas borrowed from OROCOS [@bruyninckx2001open].  -->
 
-The backbone of the framework is the Hardware Abstraction Layer, which serves as a bridge to different simulated and real robots. This abstraction is provided by the `hardware_interface::RobotHW` class; specific robot implementations have to inherit from this class. Instances of this class model hardware resources provided by the robot such as electric and hydraulic actuators and low-level sensors such as encoders and force/torque sensors. It also allows for integrating heterogeneous hardware or swapping out components transparently whether it is a real or simulated robot. Controllers are also hardware-agnostic. They specify required hardware resources to be able to operate. 
+# Packages And Functionalities
+
```
Comment (bmagyar) [inline paper.md]: please remove this newline

**Review by bmagyar — CHANGES_REQUESTED** [review]

File: paper.md (hunk 1 above)
Comment (graiola) [reply paper.md]: on papers is the 'de facto' standard

**Review by bmagyar — APPROVED** [review]

**Review by bmagyar — APPROVED** [review]

**Pull request merged**